hat/conf/play.conf

```diff
@@ -83,13 +82,18 @@ play {
       allowed = [".hubofallthings.net", ".hubat.net", ${?HAT_DOMAIN}]
     }
 
-    enabled = []
+    enabled = [
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusCounterFilter",
+        "com.github.stijndehaes.playprometheusfilters.filters.StatusAndRouteLatencyFilter",
+        "play.filters.hosts.AllowedHostsFilter",
+        "play.filters.cors.CORSFilter",
+        "play.filters.gzip.GzipFilter",
+        "org.hatdex.hat.utils.LoggingFilter",
```

Comment:
I would have that one right after the Prometheus filters like you did in dex to get better timings (since the logging filter also times the request)

Reply:
I kept the ordering as it was for the least amount of change and Im not sure if we can be garaunteed that they are run in the order specified but worth a try.

Reply:
the ordering does matter indeed